apps/site/assets/ts/shuttles/components/ShuttlesOverview.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { ReactElement, useState, useEffect } from "react";
+
+interface Props {
+  places: {
+    0: string;
+    1: string;
+  };
+}
+
+const ShuttlesOverview = ({ places }: Props): ReactElement<HTMLElement> => {
+  const [selectedDirectionId, setSelectedDirectionId] = useState("");
+  useEffect(() => {}, [selectedDirectionId]);
+
+  const directionButton = (
```

Comment:
Small point here: we capitalize any TS function that implements a UI element, whether it's a `ReactElement` or not. So in this case it's be `DirectionButton`.